infra/e2e/turing.values.yaml

```diff
@@ -34,11 +34,11 @@ turing:
             - name: treatment-1
           variants_configuration:
             control:
-              traffic: 0.85
+              traffic: 1.0
               treatment_configuration:
                 foo: bar
             treatment-1:
-              traffic: 0.15
+              traffic: 0.0
```

Comment:
@romanwozniak , I didn't manage to pick these changes up in your previous PR, but were you intending to demonstrate some treatment assignment behaviour when you introduced these configs and swapped `nop` for `proprietary` here? https://github.com/gojek/turing/blob/d12014c897a815d3526a41d390dc360e53a3a1a7/api/e2e/test/testdata/create_router_nop_logger_proprietary_exp.json.tmpl#L15
I think it's causing some odd undeterministic behaviour in the e2e tests in the CI pipeline like here:
https://github.com/gojek/turing/runs/5673171262?check_suite_focus=true#step:18:74

Reply:
@deadlycoconuts https://github.com/gojek/turing/pull/189

Reply:
Don't forget to revert this before merging @deadlycoconuts